packages/fxa-shared/test/guard/Gaurd.ts

```diff
@@ -26,10 +26,12 @@ describe('support agents', () => {
         {
           foo: {
             name: 'test foo',
+            header: 'test_foo',
```

Comment:
Nit: Same here... Filename is Gaurd instead of Guard.

Per my quick greppin' these were the only two misspelled filenames:

```sh
git ls-files | grep -i "gaurd"
    packages/fxa-shared/test/guard/AdminPanelGaurd.ts
    packages/fxa-shared/test/guard/Gaurd.ts
```

But there was one rogue "gaurd" reference in https://github.com/mozilla/fxa/blob/44956d279e972ae315addfd1f1aa19c8ec153eb5/packages/fxa-shared/test/l10n/determineLocale.ts#L58

```sh
git grep -i gaurd | cat
    packages/fxa-shared/test/l10n/determineLocale.ts:    // The spec says q-values must be between 0 and 1. We will still gaurd against bad q-values,
```